examples/http-certification/json-api/dfx.json

```diff
@@ -0,0 +1,11 @@
+{
+  "dfx": "0.15.2",
```

Comment:
```suggestion
  "dfx": "0.15.3",
```

Reply:
Thanks! I updated it also in some other places.